How to build front-end?

I did `npm install` then `npm run prod`. This error is raised. What should I do instead?
```
> @ prod /home/hamid/PhpstormProjects/horizon
> npm run production


> @ production /home/hamid/PhpstormProjects/horizon
> cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js

 95% emitting                                                                           

 ERROR  Failed to compile with 1 errors                                                                                                                                                      1:50:41 PM

This dependency was not found:

* jQuery in ./node_modules/babel-loader/lib?{"cacheDirectory":true,"presets":[["env",{"modules":false,"targets":{"browsers":["> 2%"],"uglify":true}}]]}!./node_modules/vue-loader/lib/selector.js?type=script&index=0!./resources/assets/js/components/Modals/AddTagModal.vue

To install it, you can run: npm install --save jQuery
           Asset      Size  Chunks                    Chunk Names
       js/app.js    480 kB       0  [emitted]  [big]  js/app
    /css/app.css     15 kB       0  [emitted]         js/app
   js/app.js.map    493 kB       0  [emitted]         js/app
/css/app.css.map  78 bytes       0  [emitted]         js/app

npm ERR! Linux 4.4.0-96-generic
npm ERR! argv "/home/hamid/.nvm/versions/node/v6.11.3/bin/node" "/home/hamid/.nvm/versions/node/v6.11.3/bin/npm" "run" "production"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! @ production: `cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @ production script 'cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/hamid/PhpstormProjects/horizon/npm-debug.log

npm ERR! Linux 4.4.0-96-generic
npm ERR! argv "/home/hamid/.nvm/versions/node/v6.11.3/bin/node" "/home/hamid/.nvm/versions/node/v6.11.3/bin/npm" "run" "prod"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! @ prod: `npm run production`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ prod script 'npm run production'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run production
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/hamid/PhpstormProjects/horizon/npm-debug.log
```

--- a/resources/assets/js/components/Modals/AddTagModal.vue
+++ b/resources/assets/js/components/Modals/AddTagModal.vue
@@ -1,5 +1,5 @@
 <script type="text/ecmascript-6">
-    import $ from 'jQuery'
+    import $ from 'jquery'
     import Modal from './Modal.vue'
     import Panel from '../Panels/Panel.vue'
     import Spinner from '../Loaders/Spinner.vue'

